Sources/SwiftDocC/Model/Rendering/References/TopicRenderReference.swift

```diff
@@ -106,6 +106,9 @@ public struct TopicRenderReference: RenderReference, VariantContainer {
     
     /// Author provided images that should be used to represent this page.
     public var images: [TopicImage]
+
+    /// Author provided custom metadata describing this page.
+    public var customMetadata: [String: String] = [:]
```

Comment:
Oh this is interesting! Sorry I may have missed this before.

I think this change is out-of-scope for this initial PR but we should consider it for the future. The initial goal of `@CustomMetadata` is to provide additional metadata on each individual page's JSON – expanding that to also include the custom metadata in any _references_ to that page could be a worthy enhancement but I think we should consider it separately.

(`@PageImage` was specifically designed to also be included in render references, but it's an exception from the rest of the metadata DocC considers.)

We would also need to update the RenderNode spec for render references accordingly when we make that change.

Reply:
no worries, thank you Ethan. please check now.